apps/webcomponents/src/app/components/gn-results-list/gn-results-list-multiple.sample.html

```diff
@@ -37,6 +37,12 @@
         background-color: var(--color-primary);
         margin: 30px 16px;
       }
+      .first-list {
+        --color-primary: #c40;
+      }
+      h2 {
+        color: var(--color-primary);
+      }
```

Comment:
We agree that this shouldn't be needed and that it should be possible to set two `primary-color` attributes on web components, right?

Reply:
I just illustrate a way to tweak the actual system.
Hopefully, we should improve the system so for WC, it set the css var not on html but in the current WC IMO.
It's just temporary.
Does it answer your question ?

Reply:
yes :slightly_smiling_face: